types/index.d.ts

```diff
@@ -1,4 +1,7 @@
+export as namespace DayPicker;
+
 export { default } from './DayPicker';
-export * from './common';
-export * from './props';
-export * from './utils';
+export * from './Props';
```

Comment:
The props declaration filename is still in lowercase, this seems to break some types.

Reply:
Thanks @mikaelpaj checkout v7.4.8 – should have fixed it.

Reply:
Fixed our issue, thanks for the quick reaction!